source/develop/release-management/features/virt/vm-screenshot.md

```diff
@@ -0,0 +1,83 @@
+---
+title: VM Screenshot
+category: feature
+authors: sabusale
+---
+
+# VM Screenshot
+
+## Summary
+
+Allow users to take screenshot of the VM
+
+## Owner
+
+*   Name: Saif Abu Saleh (sabusale)
+*   Email: sabusale@redhat.com
+
+## Description
+
+User can take a screenshot of running VM
```

Comment:
a running VM

and please elaborate on the use case - it's mainly intended for debugging, when you have no access (console) to the guest

Reply:
Done